solidity/truffle-examples/random-datasource/README.md

```diff
@@ -0,0 +1,66 @@
+# :wrench: :construction: Testing Oraclize's Random Example.
+
+&nbsp;
+
+This repo is to demonstrate how you would set up an Oraclize smart-contract development environment using Truffle to do most of the heavy lifting for you. Head on over to the `./test` folder to examine the javascript files that thoroughly test the smart-contract, which latter you will find in `./contracts`.
+
+## :page_with_curl:  _Instructions_
+
+Since the random datasource is currently available on the Ethereum mainnet and on all Ethereum public testnets only (Rinkeby, Kovan, Ropsten-revival); and it is *not integrated yet with private blockchains/testrpc/browser-solidity-vmmode*, you will need to **get a mnemonic passphrase and an infura key** to make the example work.
+
+**A)** Go on the infura website to get an infura key: [infura.io](https://infura.io).
+The key is a 32 HEX characters string
+
+**B)** Get a mnemonic passphrase, an easy way is to get the Metamask one:
+[metamask.io](https://metamask.io/)
+
+**C)** Create a new file `apikeys.js` or `.env` and *add the mnemonic passphrase and the infura key to it*, such as:
+
+```javascript
+// apikeys.js example
+
+module.exports = {
```

Comment:
Again, you suggest this as a format for both `apikeys` and `.env`, but for the latter this will not work.

Reply:
I was suggesting this just for apikeys

Reply:
In which case where is the bit about how to use the `.env` file for the reader of the tutorial?

Reply:
there is no reference.

Reply:
I know, do you see what I'm getting at...?

Reply:
What's going on here? Do you want me to add more details to the doc?

```
**C)** Create a new file `apikeys.js` or `.env` and *add the mnemonic passphrase and the infura key to it*
```

with the new `require('dotenv').config()` adding variables to `.env` will be enough; but, if it's not so clear, I will add an example like this in the doc:

```
// .env example 

mnemonic = 'word1 word2 ...'
infuraKey = 'f09924890213480...'
```

Reply:
Yes, that's what's going on. If you're mentioning `.env` in the `README`, those `.env` need to work. And currently, as of the reviewed commit 2ed7a8a, they do not.

Reply:
.env details added to the readme